cmsplugin_bootstrap_carousel/models_default.py

```diff
@@ -34,6 +35,14 @@ class CarouselItem(models.Model):
     button_url = models.URLField(blank=True)
     caption_content = models.TextField(blank=True, null=True)
     image = models.ImageField(upload_to="uploads/", blank=True, null=True)
```

Comment:
Would be nice to be able to override "uploads/" with a django setting, like `CAROUSEL_UPLOADS_FOLDER`